lifecycle-operator/controllers/options/keptnconfig_controller.go

```diff
@@ -51,8 +55,52 @@ func NewReconciler(client client.Client, scheme *runtime.Scheme, log logr.Logger
 	}
 }
 
+// variables needed for Keptn Gateway
+const gatewayPort = 8080
+
+var keptnGatewayDeployment = &appsv1.Deployment{
+	ObjectMeta: metav1.ObjectMeta{
+		Name:      "keptn-rest-api",
+		Namespace: "keptn-system",
```

Comment:
I'd pick this up at runtime and not enforce it. It could be the there is no ns called `keptn-system` and it's better to have the gateway running in the same ns of the lifecycle operator

Reply:
How? :)

Reply:
currently, this controller is not registered. If you register it after this line: https://github.com/keptn/lifecycle-toolkit/blob/main/lifecycle-operator/main.go#L164 
you can use the config to pickup the NS. 
I would change this from `var` to a function that accepts `ns string` as input and returns the `Deployment` object